src/common/transformations/src/transformations/op_conversions/unique_decomposition.cpp

```diff
@@ -69,6 +69,27 @@ ov::pass::UniqueDecomposition::UniqueDecomposition() {
             return false;
         }
 
+        // in case the input with the single input, there is no transformation required
+        // this separate pass needed since now there is no handler for empty tensors
+        // that should be converted to empty Constant nodes
+        // x is already of rank equal to 1 after Reshape
+        auto x_shape = x->get_input_partial_shape(0);
+        if (x_shape[0].is_static() && x_shape[0] == 1) {
```

Comment:
what if x_shape.rank() is dynamic? It looks like x_shape[0] will fail
do we support dynamic rank in this decomposition?

Reply:
we have already checked for dynamic shape in the previous line. Some cases of dynamic rank is supported when there is just one input

Reply:
It can't be dynamic because prevously we reshape it with target shape = [-1] so we know its rank for sure)